docs/en/operations/settings/settings.md

```diff
@@ -447,7 +447,17 @@ When the `input_format_tsv_enum_as_number` setting is enabled:
 ```sql
 SET input_format_tsv_enum_as_number = 1;
 INSERT INTO table_with_enum_column_for_tsv_insert FORMAT TSV 102	2;
-INSERT INTO table_with_enum_column_for_tsv_insert FORMAT TSV 103	1;
+INSERT INTO table_with_enum_column_for_tsv_insert FORMAT TSV 103	'first';
+```
+
+throws an exception.
```

Comment:
Maybe it is better to separate lines to show that the first works correctly and the second throws an exception. Or make comments for lines, or write more information after example. (this is just a thought. If you think that it is obvious, you can ignore it)

Reply:
Same for CSV